### PR TITLE
Users/subraman/createuaispec

### PR DESCRIPTION
**Pull request by mvvsubbu**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month. 10/31/2022
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month. 11/04/2022
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [x] No refresh required for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following appy to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by sabose1 — APPROVED** [review]

File: specification/databricks/resource-manager/Microsoft.Databricks/preview/2022-10-01-preview/accessconnector.json
Comment (mentat9) [inline specification/databricks/resource-manager/Microsoft.Databricks/preview/2022-10-01-preview/accessconnector.json:355]: >IdentityData [](http://example.com/codeflow?start=5&length=12)

Recommend using the ManagedServiceIdentity definition from v4 common types (managedIdentity.json) for this.

File: specification/databricks/resource-manager/Microsoft.Databricks/preview/2022-10-01-preview/accessconnector.json
Comment (mentat9) [inline specification/databricks/resource-manager/Microsoft.Databricks/preview/2022-10-01-preview/accessconnector.json:388]: >UserIdentity [](http://example.com/codeflow?start=35&length=12)

Recommend using UserAssignedIdentity from v4 common types (managedIdentity.json).

**Comment by mvvsubbu** [comment]
I am not sure why the swagger diff is done with Marketplace.json, I haven't made any breaking changes from previous commit. @lirenhe can you check and update the tags appropriately

https://github.com/Azure/azure-rest-api-specs/pull/19759/checks?check_run_id=7334352666

**Comment by mvvsubbu** [comment]
@mentat9 i have pushed the suggested changes, let me know if anything else you are looking from me. If not please go ahead and approve the PR

**Comment by mentat9** [comment]
@mvvsubbu - Please use the labels so the correct person is notified that you are ready. Just remove the ARMChangesRequested lable, and it will notify the current oncall to go back and revisit your PR.

**Comment by mvvsubbu** [comment]
@mentat9 unfortunately I don't have the ability to remove label, probably @jianyexi has the ability

**Comment by mentat9** [comment]
> @mentat9 unfortunately I don't have the ability to remove label, probably @jianyexi has the ability

It's probably because you haven't joined the Azure GH organization. I'll remove the label for you. Be sure and verify your organization membership.

**Review by lirenhe — APPROVED** [review]

**Comment by mvvsubbu** [comment]
@lirenhe why is this PR being merged? Our service capabilities are not deployed yet. I thought we are supposed to get PR approved and merge once service is deployed. Isn't it correct?

**Comment by lirenhe** [comment]
Swagger reviewers don't have a way to verify whether your service is deployed or not so we would expect service team to add 'DoNotMerge' if this PR is not ready to merge.  
If there is no such tag, we will merge the PR directly after it is approved..

**Comment by mvvsubbu** [comment]
@lirenhe thanks for letting me know. Given this is not meant to be merged. I want to get this commit reverted. What is the process for reverting?

**Comment by lirenhe** [comment]
we don't have a way to auto revert a PR, usually, we ask service team to create a new PR for reverting the previous change.
@akning-ms to comment.

**Comment by mvvsubbu** [comment]
Ok I will create a revert PR for this, please approve it.

**Comment by mvvsubbu** [comment]
@lirenhe I have created a revert PR for this https://github.com/Azure/azure-rest-api-specs/pull/20698, please help me get this merged.

**Pull request merged**
